### PR TITLE
Remove reference to tools.taskcluster.net in APIs tutorial doc

### PR DESCRIPTION
**Pull request by arku**
Bugzilla Bug: [1562752](https://bugzilla.mozilla.org/show_bug.cgi?id=1562752)

Also updates the content to reflect changes to the codebases.

### CODE REVIEW

File: ui/docs/tutorial/apis.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ followup:
 
 This section shows you how to access Taskcluster APIs programmatically.  The
 examples use JavaScript (ES6) using the JavaScript [taskcluster
-client](https://github.com/taskcluster/taskcluster-client) but apply to [other
+client](https://github.com/taskcluster/taskcluster/tree/master/clients/client) but apply to [other
 supported
 languages](/docs/manual/using/integration/libraries).
 
-Note: the graphical Taskcluster tools on `tools.taskcluster.net` are all
+Note: the graphical Taskcluster tools on `https://taskcluster-ui.herokuapp.com` are all
```
Comment (arku) [inline ui/docs/tutorial/apis.md]: @djmitche Is the reference to https://taskcluster-ui.herokuapp.com fine? We can remove this section if it is no big deal.

Comment (djmitche) [reply ui/docs/tutorial/apis.md]: I think we can just remove this paragraph -- the frontend doesn't use tc-client-web, anyway!

Comment (arku) [reply ui/docs/tutorial/apis.md]: @djmitche Done!

**Comment by djmitche** [comment]
Nice work!

**Comment by djmitche** [comment]
@arku if you have a Bugzilla account I'd be happy to assign https://bugzilla.mozilla.org/show_bug.cgi?id=1562752 to you..

**Comment by arku** [comment]
@djmitche I do. https://bugzilla.mozilla.org/user_profile?user_id=570214

Thanks!

**Pull request merged**